### ISSUE TITLE
Error in VSCode 

### ISSUE THREAD

**Issue by arnogues** [stegano/vscode-template#6]
I am actually working on the creation of react-component template and testing but there is an error in the extension : 

```
Activating extension 'yongwoo.template' failed: Cannot find module 'fs-extra'
Require stack:
- /Users/me/.vscode/extensions/yongwoo.template-1.1.0/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js.
```

The ugly work around is to type `npm install` inside the folder /Users/me/.vscode/extensions/yongwoo.template-1.1.0/

**Comment by arnogues** [comment]
I'm trying to fix the error, but I don't see any solution. 
The only solution is to remove fs-extra

**Comment by stegano** [comment]
Thanks for the bug report.
I've found a problem with the v1.1.0 version of template extensions not working properly. As a temporary measure, please check the link below and install the old version (v1.0.x). I'll fix it soon.
- https://code.visualstudio.com/updates/v1_30#_install-previous-versions

@all-contributors please add @arnogues for bug

**Comment by arnogues** [comment]
> Thanks for the bug report.
> I've found a problem with the v1.1.0 version of template extensions not working properly. As a temporary measure, please check the link below and install the old version (v1.0.x). I'll fix it soon.
> 
> * https://code.visualstudio.com/updates/v1_30#_install-previous-versions
> 
> @all-contributors please add @arnogues for bug

I d'un `npm install` where the extension is installés and it work properly.

**Comment by stegano** [comment]
@arnogues 
This issue was caused by not including dependency libraries in the extension package because the `.vscodeignore` file contains the node_modules path.

since v1.1.0 version, source files were bundled using 'webpack' and there was a problem loading templates that existed inside the extension, so we changed and republish them in the same way as before.

This issue is resolved in v1.1.4.
Thanks for reporting the bug
